R/zchunk_L1233.Elec_water.R

```diff
@@ -1,27 +1,26 @@
 #' module_water_L1233.Elec_water
 #'
-#' Briefly describe what this chunk does.
+#' Water consumption and withdrawals for electricity.
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L1233.out_EJ_R_elec_F_tech_Yh_cool}, \code{L1233.in_EJ_R_elec_F_tech_Yh_cool}, \code{L1233.wdraw_km3_R_elec}, \code{L1233.wcons_km3_R_elec}, \code{L1233.shrwt_R_elec_cool_Yf}. The corresponding file in the
 #' original data system was \code{L1233.Elec_water.R} (water level1).
-#' @details Describe in detail what this chunk does.
+#' @details Categorizes electiricty generating technologies by water type, and computes water withdrawals and consumption.
 #' @importFrom assertthat assert_that
 #' @importFrom dplyr filter mutate select
 #' @importFrom tidyr gather spread
-#' @author YourInitials CurrentMonthName 2017
-#' @export
```

Comment:
I think I figured it out - I think the red text with the minus before the poundsign indicates that line was removed. Looking at the full diff on this script and on mine yesterday, right below the red is a green line with a plus out front and the correct author information. Perhaps this indicates that the blank author and export lines were removed, and the correct author info line was added?